tutel/impls/moe_layer.py

```diff
@@ -73,6 +74,8 @@ def __init__(
         input_dropout_p = kwargs.get('input_dropout_p', 0)
         self.input_dropout = torch.nn.Dropout(p=input_dropout_p) if input_dropout_p else None
 
+        self.a2a_ffn_overlap_degree = a2a_ffn_overlap_degree
+
```

Comment:
In order not to change the original usage of TopKGate, better to set `a2a_ffn_overlap_degree=1` by default or using `kwargs.get('a2a_ffn_overlap_degree', 1)` to access the value.

Reply:
Fixed.